scheduler-plugin/fluence/cmd/main.go

```diff
@@ -1,25 +1,24 @@
 package main
 
 import (
-	"fmt"
 	"flag"
-	"net"
-	"google.golang.org/grpc/keepalive"
+	"fmt"
```

Comment:
Does the import order matter?

Reply:
They do - I usually let my editor (VSCode) determine the order (details here https://teamtreehouse.com/community/does-it-matter-what-order-imports-go) and if there is something that breaks based on that convention I’d expect the library not to work (although this hasn’t happened to me yet!)